integrations/gateway-api-inference-extension/documentation.yaml

```diff
@@ -0,0 +1,44 @@
+exporter_type: included
+app_name_short: Inference Extension
+app_name: {{app_name_short}}
+app_site_name: {{app_name_short}}
+app_site_url: https://github.com/kubernetes-sigs/gateway-api-inference-extension/
+exporter_name: the {{app_name_short}} exporter
+exporter_pkg_name: inference-extension
+exporter_repo_url: https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/site-src/guides/metrics.md
+additional_prereq_info: |
+  {{app_site_name}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. To verify that {{exporter_name}} is emitting
+  metrics on the expected endpoints, do the following:
+
+  1. Set up port forwarding by using the following command:
+
+  <pre class="devsite-click-to-copy">
+  kubectl -n {{namespace_name}} port-forward {{pod_name}} 9090
+  </pre>
+
+  2. Access the endpoint `localhost:9090/metrics` by using the `curl` utility in another terminal session
+  with token in command following the instruction: https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/site-src/guides/metrics.md#scrape-metrics
+dashboard_available: true
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: inference-extension
+    labels:
+      app.kubernetes.io/name: inference-gateway
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+    - port: metrics
+      scheme: http
+      interval: 5s
+      path: /metrics
+    selector:
+      matchLabels:
+        app: inference-gateway-ext-proc
+additional_podmonitoring_info: |
+  Ensure that the values of the `port` and `matchLabels` fields match those of the {{app_name_short}} pods you want to monitor.
+sample_promql_query: inference_model_request_total
```

Comment:
If you look at the others, we typically do `inference_model_request_total{cluster="{{cluster_name}}", namespace="{{namespace_name}}"}` to give a sample to test your specific instance when you can have multiple instances up in the same project.